### PR TITLE
Adds TracedScheduledExecutorService

### PR DESCRIPTION
**Pull request by jam01**
Adds TracedScheduledExecutorService and tests

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   <groupId>io.opentracing.contrib</groupId>
   <artifactId>opentracing-concurrent</artifactId>
-  <version>0.1.1-SNAPSHOT</version>
+  <version>0.2.0-SNAPSHOT</version>
```
Comment (objectiser) [inline pom.xml]: Can you revert this, as the release procedure automatically updates the versions.

File: src/test/java/io/opentracing/contrib/concurrent/TracedScheduledExecutorServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+		MockSpan parentSpan = mockTracer.buildSpan("foo").startManual();
+		mockTracer.scopeManager().activate(parentSpan, true);
+		executorService.scheduleAtFixedRate(new TestRunnable(), 0, 300, TimeUnit.MILLISECONDS);
+
+		countDownLatch.await();
+
+		assertParentSpan(parentSpan);
+		assertEquals(2, mockTracer.finishedSpans().size());
+	}
```
Comment (objectiser) [inline src/test/java/io/opentracing/contrib/concurrent/TracedScheduledExecutorServiceTest.java]: Should shutdown the executor service as it is repetitive task - same for following test.

**Review by objectiser — APPROVED** [review]

**Comment by objectiser** [comment]
@pavolloffay ok to merge and release?

**Review by pavolloffay — APPROVED** [review]

**Comment by pavolloffay** [comment]
thanks @jam01!

**Comment by objectiser** [comment]
@jam01 Version 0.2.0 should be available soon, so can be used from https://github.com/opentracing-contrib/java-agent/pull/41

**Pull request merged**